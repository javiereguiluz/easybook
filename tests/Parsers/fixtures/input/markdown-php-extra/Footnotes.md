This is the first paragraph.[^first]

[^first]:  This is the first note.

* List item one.[^second]
* List item two.[^third]

[^third]: This is the third note, defined out of order.
[^second]: This is the second note.
[^fourth]: This is the fourth note.

# Header[^fourth]

Some paragraph with a footnote[^1], and another[^2].

[^1]: Content for fifth footnote.
[^2]: Content for sixth footnote spaning on 
    three lines, with some span-level markup like
    _emphasis_, a [link][].

[link]: http://www.michelf.com/

Another paragraph with a named footnote[^fn-name].

[^fn-name]:
    Footnote beginning on the line next to the marker.

This paragraph should not have a footnote marker since 
the footnote is undefined.[^3]

This paragraph should not have a footnote marker since 
the footnote has already been used before.[^1]

This paragraph links to a footnote with plenty of 
block-level content.[^block]

[^block]:
	Paragraph.
	
	*   List item
	
	> Blockquote
	
	    Code block

This paragraph host the footnote reference within a 
footnote test[^reference].

[^reference]:
	This footnote has a footnote of its own.[^nested]

[^nested]:
	This footnote should appear even though as it is refered
	from another footnote. But [^reference] should be litteral
	since the footnote with that name has already been used.

 - - -

Testing unusual footnote name[^1$^!"'].

[^1$^!"']: Haha!
