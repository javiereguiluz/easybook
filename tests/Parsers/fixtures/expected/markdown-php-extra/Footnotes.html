<p>This is the first paragraph.<sup id="fnref:first"><a href="#fn:first" class="footnote-ref">1</a></sup></p>

<ul>
<li>List item one.<sup id="fnref:second"><a href="#fn:second" class="footnote-ref">2</a></sup></li>
<li>List item two.<sup id="fnref:third"><a href="#fn:third" class="footnote-ref">3</a></sup></li>
</ul>

<h1>Header<sup id="fnref:fourth"><a href="#fn:fourth" class="footnote-ref">4</a></sup></h1>

<p>Some paragraph with a footnote<sup id="fnref:1"><a href="#fn:1" class="footnote-ref">5</a></sup>, and another<sup id="fnref:2"><a href="#fn:2" class="footnote-ref">6</a></sup>.</p>

<p>Another paragraph with a named footnote<sup id="fnref:fn-name"><a href="#fn:fn-name" class="footnote-ref">7</a></sup>.</p>

<p>This paragraph should not have a footnote marker since 
the footnote is undefined.[^3]</p>

<p>This paragraph should not have a footnote marker since 
the footnote has already been used before.<sup id="fnref2:1"><a href="#fn:1" class="footnote-ref">5</a></sup></p>

<p>This paragraph links to a footnote with plenty of 
block-level content.<sup id="fnref:block"><a href="#fn:block" class="footnote-ref">8</a></sup></p>

<p>This paragraph host the footnote reference within a 
footnote test<sup id="fnref:reference"><a href="#fn:reference" class="footnote-ref">9</a></sup>.</p>

<hr />

<p>Testing unusual footnote name<sup id="fnref:1$^!&quot;'"><a href="#fn:1$^!&quot;'" class="footnote-ref">10</a></sup>.</p>

<div class="footnotes">
<hr />
<ol>

<li id="fn:first">
<p>This is the first note.&#160;<a href="#fnref:first" class="footnote-backref">&#8617;&#xFE0E;</a></p>
</li>

<li id="fn:second">
<p>This is the second note.&#160;<a href="#fnref:second" class="footnote-backref">&#8617;&#xFE0E;</a></p>
</li>

<li id="fn:third">
<p>This is the third note, defined out of order.&#160;<a href="#fnref:third" class="footnote-backref">&#8617;&#xFE0E;</a></p>
</li>

<li id="fn:fourth">
<p>This is the fourth note.&#160;<a href="#fnref:fourth" class="footnote-backref">&#8617;&#xFE0E;</a></p>
</li>

<li id="fn:1">
<p>Content for fifth footnote.&#160;<a href="#fnref:1" class="footnote-backref">&#8617;&#xFE0E;</a> <a href="#fnref2:1" class="footnote-backref">&#8617;&#xFE0E;</a></p>
</li>

<li id="fn:2">
<p>Content for sixth footnote spaning on 
three lines, with some span-level markup like
<em>emphasis</em>, a <a href="http://www.michelf.com/">link</a>.&#160;<a href="#fnref:2" class="footnote-backref">&#8617;&#xFE0E;</a></p>
</li>

<li id="fn:fn-name">
<p>Footnote beginning on the line next to the marker.&#160;<a href="#fnref:fn-name" class="footnote-backref">&#8617;&#xFE0E;</a></p>
</li>

<li id="fn:block">
<p>Paragraph.</p>

<ul>
<li>List item</li>
</ul>

<blockquote>
  <p>Blockquote</p>
</blockquote>

<pre><code>Code block
</code></pre>

<p><a href="#fnref:block" class="footnote-backref">&#8617;&#xFE0E;</a></p>
</li>

<li id="fn:reference">
<p>This footnote has a footnote of its own.<sup id="fnref:nested"><a href="#fn:nested" class="footnote-ref">11</a></sup>&#160;<a href="#fnref:reference" class="footnote-backref">&#8617;&#xFE0E;</a></p>
</li>

<li id="fn:1$^!&quot;'">
<p>Haha!&#160;<a href="#fnref:1$^!&quot;'" class="footnote-backref">&#8617;&#xFE0E;</a></p>
</li>

<li id="fn:nested">
<p>This footnote should appear even though as it is refered
from another footnote. But [^reference] should be litteral
since the footnote with that name has already been used.&#160;<a href="#fnref:nested" class="footnote-backref">&#8617;&#xFE0E;</a></p>
</li>

</ol>
</div>
